### DISCUSSION TITLE
Regarding mypdns.org

### DISCUSSION THREAD

**Discussion by spirillen** [upptime/upptime#663] [Q&A]
Regarding https://github.com/thomasmerz/upptime/issues/1502

I think you might have been a victim for my firewall.....

And this is not yours fault, but some do steal all my work by using crawlers and other bots and I do know I have been blacklisting github GHA before (fail2ban and nginx rules)

Do you have any suggestion for how we might be able to ensure a safe passage?

**Comment by AnandChowdhary** [comment]
Since your URLs are private, maybe you can add an extra query parameter like https://example.com?api_key=upptime_4u29830i1 and allow requests from all IP addresses? Since we use GitHub Actions, if you block all GitHub IP addressses, Upptime will also not work.

**Reply by spirillen** [reply]
Well changing the url/domain won't fix it as the firewall rules blocks all ports from saddr for a given time, the only solution might be a reverse proxy, which I do not have the possibility to setup for now
